build.gradle

```diff
@@ -8,7 +8,13 @@ buildscript {
 }
 
 plugins {
-    id 'nebula.netflixoss' version '3.4.0'
+//    id 'nebula.netflixoss' version '3.4.0'
+    id 'nebula.maven-publish' version '4.9.1'
+    id 'nebula.release' version '4.1.0'
+    id 'nebula.dependency-lock' version '4.3.0'
+    id 'nebula.info' version '3.2.1'
+    id 'idea'
+    id 'eclipse'
```

Comment:
The netflixoss plugin is something like a meta-plugin that includes a lot of other nebula plugins, I just included manually the useful ones except the one they use to publish their own artifacts